Description: Disables support for the AWS credential service (AWS library not in Debian)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed

--- a/pom.xml
+++ b/pom.xml
@@ -148,6 +148,7 @@
           </compilerArgs>
           <excludes>
             <exclude>**/Windows*</exclude>
+            <exclude>**/Aws**</exclude>
           </excludes>
         </configuration>
       </plugin>
